lib/ansible/plugins/action/__init__.py

```diff
@@ -421,7 +421,7 @@ def _make_tmp_path(self, remote_user=None):
             self._cleanup_remote_tmp = True
 
         try:
-            stdout_parts = result['stdout'].strip().split('%s=' % basefile, 1)
+            stdout_parts = result['stdout'].strip().replace('\r', '').replace('\n', '').split('%s=' % basefile, 1)
```

Comment:
while this might fix your specific issue it might create many issues for others

a better approach is making sure that we always have an ample buffer size when executing commands\

cc @jborean93

Reply:
Ok, thanks. 
Is there a way to resolve this issue with something like my own local plugin which overrides ActionBase and patches this behavior? Can I write such plugin only for the base class and get right inheritance of other descendants?